aws/validators.go

```diff
@@ -82,6 +82,34 @@ func validateTypeStringNullableFloat(v interface{}, k string) (ws []string, es [
 	return
 }
 
+func validateTransferServerID(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
Nit: These custom validation functions can be replaced with `ValidateFunc: validation.StringMatch(regexp.MustCompile(), "message")`. We'll be doing a passthrough of cleaning up all the extraneous custom validation functions after the Terraform 0.12 provider SDK is merged as it contains some very helpful new functionality in this regard. 👍